src/eatery/campus_eatery.py

```diff
@@ -0,0 +1,89 @@
+import requests
+
+from src.constants import (
+    STATIC_MENUS_URL,
+    TRILLIUM_ID,
+)
+from src.types import (
+    CampusEateryType,
+    OperatingHoursType
+)
+from src.eatery.common_eatery import (
+    get_image_url,
+    parse_campus_area,
+    parse_coordinates,
+    parse_dining_items,
+    parse_eatery_type,
+    parse_events,
+    parse_payment_methods,
+    resolve_id
+)
+
+def parse_eatery(data_json, campus_eateries):
+  """Parses a Cornell Dining json dictionary.
+
+  Fills the campus_eateries dictionary with objects of type CampusEateryType.
+
+  Args:
+      data_json (dict): a valid dictionary from the Cornell Dining json
+      campus_eateries (dict): a dictionary to fill with campus eateries
+  """
+  for eatery in data_json['data']['eateries']:
+    eatery_id = eatery.get('id', resolve_id(eatery))
+    dining_items = get_trillium_menu() if eatery_id == TRILLIUM_ID else parse_dining_items(eatery)
+    phone = eatery.get('contactPhone', 'N/A')
+    phone = phone if phone else 'N/A'  # handle None values
+
+    new_eatery = CampusEateryType(
+        about=eatery.get('about', ''),
+        campus_area=parse_campus_area(eatery),
+        coordinates=parse_coordinates(eatery),
+        eatery_type=parse_eatery_type(eatery),
+        id=eatery_id,
+        image_url=get_image_url(eatery.get('slug', '')),
+        location=eatery.get('location', ''),
+        name=eatery.get('name', ''),
+        name_short=eatery.get('nameshort', ''),
+        operating_hours=parse_operating_hours(eatery, dining_items),
+        payment_methods=parse_payment_methods(eatery['payMethods']),
+        phone=phone,
+        slug=eatery.get('slug', '')
+    )
+    campus_eateries[new_eatery.id] = new_eatery
+
+def get_trillium_menu():
+  """Gets the Trillium menu.
+
+  Returns the Trillium dining items (using parse_dining_items) from the static json source
+  for menus.
+  """
+  static_json = requests.get(STATIC_MENUS_URL).json()
+  return parse_dining_items(static_json['Trillium'][0])
+
+
```

Comment:
Single or double lines between function declarations? @meganle

Reply:
Let's keep things consistent so single lines seems to be the norm